config-v3-documentation

```diff
@@ -85,6 +86,25 @@ judge_needs_out=0
 # Set to 1 if judge needs input/output
 # Defaults to 1
 
+tokens_ignore_newlines=0
+# Only for out_check=tokens
+# If set to 1, newline characters will be ignored when checking the output,
+# as if they were any other whitespace characters
+# By default, newline characters are only ignored at the end of the file
+
+tokens_ignore_case=0
+# Only for out_check=tokens
+# If set to 1, ASCII characters will be compared in a case-insensitive manner
+
+tokens_float_rel_error=0.00001
+tokens_float_abs_error=1e-30
+# Only for out_check=tokens
+# When these options are specified, floating-point numbers
+# will be parsed and compared with a given error margin
+# Any tokens that can't be parsed as a float will be compared character-by-character
+# If used, both of these options must be specified
+# Set both options to X to disable
```

Comment:
I don't really know about the `X`. We use it in different context with a different meaning. I would much prefer either `@` value or empty to disable.

Reply:
I thought it was kind of similar to `max_submissions`. I've changed the unset value to the empty string.

Reply:
> I thought it was kind of similar to `max_submissions`. I've changed the unset value to the empty string.

Something like `@unlimited` could be better, but that's up to another PR.

Reply:
> I thought it was kind of similar to `max_submissions`. I've changed the unset value to the empty string.

Can you also update the documentation?

Reply:
Fixed.